compiler-plugin/src/main/kotlin/arrow/meta/plugins/union/UnionsPlugin.kt

```diff
@@ -203,7 +200,7 @@ private fun IrUtils.nullableConversionCall(typeArgument: IrType?, argument: IrEx
  * val willBeInt: Int? = proof() //ok
  * ```
  */
-private fun Diagnostic.suppressTypeMismatchOnNullableReceivers(): Boolean =
+fun Diagnostic.suppressTypeMismatchOnNullableReceivers(): Boolean =
```

Comment:
🔥

Reply:
@nomisRev Should I change something? Or what does the fire mean 😄 ?

Reply:
No, all good :D I just love the code re-use between compiler and ide plugins.